index.js

```diff
@@ -1,92 +1,114 @@
-var fs    = require('fs'),
+'use strict';
+
+const fs  = require('fs'),
     path  = require('path'),
     spawn = require('child_process').spawn,
+    async = require('async'),
     temp  = require('temp');
 
 exports.generateFdf = function(data) {
   var header, body, footer, dataKeys;
 
-  // We should really come up with a new way of making FDF files, this is doing a lot of buffer instantiation.
-
-  header = new Buffer("%FDF-1.2\n\u00e2\u00e3\u00cf\u00d3\n1 0 obj \n<<\n/FDF \n<<\n/Fields [\n");
+  header = new Buffer('%FDF-1.2\n\u00e2\u00e3\u00cf\u00d3\n1 0 obj \n<<\n/FDF \n<<\n/Fields [\n');
 
-  footer = new Buffer("]\n>>\n>>\nendobj \ntrailer\n\n<<\n/Root 1 0 R\n>>\n%%EOF\n");
+  footer = new Buffer(']\n>>\n>>\nendobj \ntrailer\n\n<<\n/Root 1 0 R\n>>\n%%EOF\n');
 
   dataKeys = Object.keys(data);
 
   body = new Buffer([]);
 
   for(var i=0; i<dataKeys.length; i++) {
     var name = dataKeys[i].toString();
-    var value = data[name].toString().replace("\r\n","\r");
+    var value = data[name].toString().replace('\r\n','\r');
 
-    body = Buffer.concat([ body, new Buffer("<<\n/T (" + name + ")\n/V (" + value + ")\n>>\n") ]);
+    body = Buffer.concat([ body, new Buffer('<<\n/T (' + name + ')\n/V (' + value + ')\n>>\n') ]);
   }
 
   var fdf =  Buffer.concat([ header, body, footer ]);
   return fdf;
-}
+};
 
 exports.generatePdf = function(data, templatePath, extendArgs, callback) {
   var tempNameResult = temp.path({suffix: '.pdf'}),
       pdfPath        = isAbsolute(templatePath) ? templatePath : path.join(__dirname, templatePath);
 
+  [extendArgs, callback] = normalizeArgs(extendArgs, callback);
```

Comment:
Don't want to introduce a build step to this module. Can reduce features to v4 LTS.